core/docs/API-Changelog-0.1.0.txt

```diff
@@ -0,0 +1,424 @@
+---------------------------------
```

Comment:
We can put this in a changelog directory, and then keep adding a file for each release.